### ISSUE TITLE
acc decrease from 0.8xxx to 0.5xxx when normalized in FashionMNIST

### ISSUE THREAD

**Issue by vxgu86** [jeremyfix/deeplearning-lectures#15]
I test the FullyConnectedRegularized with normalized set to True, but the acc decrease from 0.8xxx to 0.5xxx when normalized in FashionMNIST.
You said that the FashionMNIST dataset is already normalized, but the normalized after normalized would have no effect from math perspective, am I right?

**Comment by vxgu86** [comment]
I am only testing the val metrics acc after each epoch and the test acc,
from the code above, I did not compute the mean and std with augmented dataset，right? it's loaded from **train_dataset** without augmented.

what I get is that:
for linearNet, 
the best test acc 0.8420 is from the no augment and no normalized version
with augment, the test acc is 0.7322, 
with augment and normalization, the test acc is 0.4602.
it's really big margin among the three accs,  may I ask, what's the problem?

**Comment by vxgu86** [comment]
sorry I did get some trouble from the COVID-19.
but I remember that you mentioned somewhere that the dataset has already been normalized.
then the normalization on train dataset should have no effect, neither on the test and valid dataset.
is this thought wrong?

**Comment by vxgu86** [comment]
@jeremyfix hello  are you there?

**Comment by jeremyfix** [comment]
@vxgu86 More or less there, from time to time;

If I'm not incorrect, a quite glance at a [comment above](https://github.com/jeremyfix/deeplearning-lectures/issues/15#issuecomment-948902641) and I'm not sure that normalization does have a significant effect indeed;  That we do not get the sames results from runs to runs is true, but that's because of the stochasticity in the initialization;

Now, an effect that I observed in the past was like the one in the [tutorial on keras](https://teaching.pages.centralesupelec.fr/deeplearning-lectures-build/00-keras-mnist.html) . You can compare the metrics between the image "Logistic regression without normalization of the input" and "Logistic regression with input standardization" and you will see that we have a strong effect of standardization on the mean and standard deviation of the metrics even on the training set. On the specific cases of the figure, this is actually a problem because the problem at hand is convex and different losses on the training set is not at all expected. All that mean is that the optimization problem is not solved the appropriate way and actually explained by the fact that, with input not standardized, the optimization problem, although convex, might be bad conditioned (the slope of the loss might be really steep in some directions compared to others);

**Comment by jeremyfix** [comment]
@vxgu86  Should we consider your question answered ?

**Comment by vxgu86** [comment]
sorry I got it, thanks
